calicoctl/resourcemgr/resourcemgr.go

```diff
@@ -21,14 +21,15 @@ import (
 	"reflect"
 	"strings"
 
-	"io/ioutil"
+	"io"
```

Comment:
nit:  I realize this was already here, but we should probably just group together all of the native golang imports rather than having them split up across four groups.